docs/docs/tutorials/index.mdx

```diff
@@ -15,6 +15,7 @@ New to LangChain or to LLM app development in general? Read this material to qui
 ## Working with external knowledge
 - [Build a Retrieval Augmented Generation (RAG) Application](/docs/tutorials/rag)
 - [Build a Conversational RAG Application](/docs/tutorials/qa_chat_history)
+- [Build a Tech Support Bot from an existing Knowledge Base](/docs/tutorials/graph_vectorstore)
```

Comment:
Possible titles to get RAG and links in:

`Build a Tech Support RAG Application with Content Links` (Content Links or Hyperlinks or something like that perhaps)?